OpenFoodFactsPower.user.js

```diff
@@ -916,7 +917,35 @@ content: " — ";
 
     }
 
-	//Copy data from the language specific ingredients_text to the ingredients_text in the hidden form so it can be poassed to the analyser
+    //Copy data from the list textarea to the ingredients_text in the hidden form so it can be passed to the analyser
+    //As the list can contain different languages we take the language from the textarea
+	function CopyListData(_code, lang){
+		console.log("Lang:" + lang);
+		var cd = $("#i" + _code).val()
+		console.log("Language Text:"+cd);
+
+        //Here we have to manipulate the language for regional languages
+        if(lang === 'ca'){lang = 'es-ca';} //Catalan
+        if(lang === 'en'){
+            if(pageLanguage === 'en'){
+                lang = 'uk';//English
+            }
+            else
+            {
+                lang = pageLanguage + '-en'; //English from source language page
+            }
+        }
+
+		//As target language can be different from the page language we have to create the full URL
+		var URL = "http:/" + lang + ".openfoodfacts.org/cgi/test_ingredients_analysis.pl";
```

Comment:
```suggestion
		var URL = "//" + lang + ".openfoodfacts.org/cgi/test_ingredients_analysis.pl";
```
Firefox (at least) is giving an alert if you use http:/. Two slashes should work.